wordpress-mobile/WordPress-iOS
# Issue/6157 - Shared pod update fixes table header/footer layout issues

Fixes #6157

This pulls in `WordPress-iOS-Shared 0.7.0` to not set a custom font with size for either table header or footer views. This also deletes lingering imports of the now removed `WPTableViewSectionHeaderFooterView`.

To test:
- `pod install`
- Build, make sure no warnings or errors
  - Ignore the "Update Project Settings" if you see it, fixed in https://github.com/wordpress-mobile/WordPress-iOS/pull/6180
- On iPad Air or Pro, Open the Me tab, Help & Support
  - Ensure the footer labels are aligned and lay out correctly. 

Needs review: @frosty can you give it a go?
